docs/08-bootstrapping-kubernetes-controllers.md

```diff
@@ -82,14 +82,13 @@ ExecStart=/usr/local/bin/kube-apiserver \\
   --authorization-mode=Node,RBAC \\
   --bind-address=0.0.0.0 \\
   --client-ca-file=/var/lib/kubernetes/ca.pem \\
-  --enable-admission-plugins=Initializers,NamespaceLifecycle,NodeRestriction,LimitRanger,ServiceAccount,DefaultStorageClass,ResourceQuota \\
-  --enable-swagger-ui=true \\
```

Comment:
@nefigtut - With this removal, what do you have in mind? Documentation makes this flag seem relevant today - though maybe it isn't entirely applicable here.

Reply:
@Knaledge - when run with `--enable-swagger-ui=true` `kube-apiserver` says:

`Flag --enable-swagger-ui has been deprecated, swagger 1.2 support has been removed`

so i've removed this flag.